chessengine/bitboard.py

```diff
@@ -617,6 +606,48 @@ def alpha_beta_search(
                 beta = min(beta, value)
             return value
 
+    def get_input(self, prompt: str) -> str:
+        """Wrapper for input to allow testing"""
+        return input(prompt).strip()
```

Comment:
I'm not sure about the purpose of this wrapper. Do you plan to extend it's functionality? I think we could do without it since it doesn't do much at the moment.

Reply:
This was entirely for test mocking purposes (https://github.com/amcintosh/chessengine/blob/issue-37-refactor-play-play-pvp/tests/test_bitboard.py#L76-L78). `input()` cannot be mocked directly, but the wrapper can.

Reply:
Oh, I see. Everything looks good then, could you run `black` once on your changes to get the linting to pass? Then I'll be able to merge.

Reply:
Updated!

Reply:
Not really sure why it's failing. Black seesm happy locally:

```shell
$ black chessengine/bitboard.py 
All done! ✨ 🍰 ✨
1 file would be left unchanged.

$ black tests/test_bitboard.py 
All done! ✨ 🍰 ✨
1 file would be left unchanged.
```

Reply:
Can you try updating your `black` version once? I just upgraded to version `22.10.0` locally, and it does tell me to reformat both those files. Once this is done, I think I'll pin `black`'s version to `22.10.0` in the GitHub action for consistency since `black` seems to do this often.

Reply:
Ah. That did it. Thanks